src/plugins/intel_gpu/src/kernel_selector/core/cl_kernels/fully_connected_gpu_imad.cl

```diff
@@ -118,6 +138,96 @@ KERNEL(fully_connected_gpu_imad)(
         }
     }
 
+
+    // Main calculation cycle by IFM (leftovers)
+#if HAS_IFM_LEFTOVERS
+    // Loading weights
+    MAKE_VECTOR_TYPE(int, SIMD_SIZE) weights_data[TILE_OFM];
+    __attribute__((opencl_unroll_hint))
+    for (uint of_idx = 0; of_idx < TILE_OFM; of_idx++) {
+        weights_data[of_idx] = 0;
+        __attribute__((opencl_unroll_hint))
+        for (uint if_idx = 0; if_idx < SIMD_SIZE; if_idx++) {
+            char4 temp_weights = 0;
```

Comment:
how about initialize more precisely, 
char4 temp_eights = (char4)0;

Reply:
Thanks, I'll fix it.